### PR TITLE
Allows relative paths for kubeconfig and argsfile

### PR DESCRIPTION
**Pull request by srm09**
Fixes: #178

### CODE REVIEW

File: util/path.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+package util
+
+import (
+	"github.com/mitchellh/go-homedir"
+)
+
+// ExpandPath converts the file path to include the home directory when prefixed with `~`.
+func ExpandPath(path string) (string, error) {
+	return homedir.Expand(path)
+}
```
Comment (vladimirvivien) [inline util/path.go:13]: Ok, so I checked and I don't think we need to import this, less baggage. 

The one thing that `homedir.Exapand()` seems to support is expanding path with tilde (`~/go`). Could we achieve the same thing with something like this:
```go
func ExpandPath(path string)(string,error) {
    home, _ := os.UserHomeDir()
    if path[0] == '~' {
        filepath.Join(home, path[1:])
    }
    return path
}
```

Mitchell's code is doing something similar. It only fallsback to other methods if the user home env is not set. You can see Go is already taking care of something similar
https://golang.org/src/os/file.go?s=14399:14433#L479 

Let me know what you think

**Review by vladimirvivien — COMMENTED** [review]
Take a look.  I think we can avoid that import.

File: util/path.go (hunk 1 above)
Comment (srm09) [reply util/path.go:13]: Initially, I tried to do this thing. Then I wasn't sure if that would work or not always, hence used the library instead after some searching. I will revert this, and use the simpler implementation like you have mentioned above.

File: util/path.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+package util
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// ExpandPath converts the file path to include the home directory when prefixed with `~`.
+func ExpandPath(path string) string {
+	home, _ := os.UserHomeDir()
```
Comment (vladimirvivien) [inline util/path.go:13]: We need the error check.

**Review by vladimirvivien — COMMENTED** [review]
Looks good, but we need the error check If os.UserHomeDir fails.  Sorry should have been clearer in my prev review.

**Review by vladimirvivien — APPROVED** [review]
LGTM.  Thanks for adding the test.

**Pull request merged**